packages/rollup-plugin-styles/src/loaders/postcss/import/index.ts

```
@@ -0,0 +1,153 @@
+    const resolve = options.resolve ?? resolveDefault;
+    const alias = options.alias ?? {};
+    const extensions = options.extensions ?? extensionsDefault;
+
+    return {
+        async Once(css, { result: res }) {
+            if (!css.source?.input.file) {
+                return;
+            }
+
+            const options_: Result["opts"] = { ...res.opts };
+            delete options_.map;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)